src/compojure_throttle/core.clj

```diff
@@ -1,20 +1,29 @@
 (ns compojure-throttle.core
-  (:require [clojure.core.cache :as cache]
-            [environ.core :refer [env]]
+  (:require [clj.ip :refer [compile]]
+            [clj-time.core :as core-time]
             [clj-time.local :as local-time]
-            [clj-time.core  :as core-time]))
+            [clojure.core.cache :as cache]
+            [environ.core :refer [env]]))
 
 (def ^:private defaults
-  {:service-compojure-throttle-enabled "true"
-   :service-compojure-throttle-ttl    1000
-   :service-compojure-throttle-tokens 3
+  {:service-compojure-throttle-enabled       "true"
+   :service-compojure-throttle-lax-ips       "127.0.0.1/32"
+   :service-compojure-throttle-ttl           1000
+   :service-compojure-throttle-tokens        3
    :service-compojure-throttle-response-code 429})
 
 (defn enabled?
   []
   (boolean (Boolean/valueOf
-            (or (env :service-compojure-throttle-enabled)
-                (defaults :service-compojure-throttle-enabled)))))
+             (or (env :service-compojure-throttle-enabled)
+                 (defaults :service-compojure-throttle-enabled)))))
+
+(defn- ip-subnet
```

Comment:
Since the underlying setting is about `lax-ips`, I'd like to suggest a different name for this so it's less general. Could be really simple: how's `lax-ip-subnet`?